src/languages/less.js

```diff
@@ -124,17 +131,26 @@ export default function(hljs) {
       IDENT_MODE('selector-class', '\\.' + INTERP_IDENT_RE, 0),
       IDENT_MODE('selector-tag',  '&', 0),
       {className: 'selector-attr', begin: '\\[', end: '\\]'},
-      {className: 'selector-pseudo', begin: /:(:)?[a-zA-Z0-9\_\-\+\(\)"'.]+/},
+      {className: 'selector-pseudo', begin: /:(:)?[a-zA-Z0-9\_\-\+"'.]+/},
```

Comment:
Can this regex also be shared? The CSS grammar is using this on [line 91](https://github.com/highlightjs/highlight.js/pull/2425/files#diff-335a561957078af90e8fbf6c8dfae8a7R91).

Reply:
Agree, maybe I should have specified this is work in progress. :-)  Pulling these tiny things out is likely the last step since they are the most likely to have tiny differences hidden.